src/components/structural/Textures.js

```diff
@@ -0,0 +1,30 @@
+let TexturePack = [
+    {bark : "https://upload.wikimedia.org/wikipedia/commons/4/41/Cedrus_libani_Bark.jpg"},
+    {blank : "https://upload.wikimedia.org/wikipedia/commons/e/eb/Blank.jpg"},  
+    {brick : "https://upload.wikimedia.org/wikipedia/commons/f/f0/Red_brick_texture.jpg"},
+    {cement : "https://upload.wikimedia.org/wikipedia/commons/1/1e/Cement_texture.jpg"},
+    {cobblestone : "https://upload.wikimedia.org/wikipedia/commons/b/be/Ancient_road_surface.jpg"},
+    {duck : "https://upload.wikimedia.org/wikipedia/commons/2/2a/Duck-293474_white_background.jpg"},
+    {error : "https://img.icons8.com/cotton/2x/error--v1.png"},
+    {floral : "https://upload.wikimedia.org/wikipedia/commons/thumb/5/51/Original_William_Morris%27s_patterns%2C_digitally_enhanced_by_rawpixel_00031.jpg/720px-Original_William_Morris%27s_patterns%2C_digitally_enhanced_by_rawpixel_00031.jpg"},
+    {grass : "https://upload.wikimedia.org/wikipedia/commons/a/a6/Grass_01.jpg"},
+    {ground : "https://upload.wikimedia.org/wikipedia/commons/4/4b/Ground_dirt_sand.jpg"},
+    {illusion : "https://img-s-msn-com.akamaized.net/tenant/amp/entityid/AAHAKHD.img?h=315&w=600&m=6&q=60&o=t&l=f&f=jpg&x=294&y=260"},
+    {lava : "https://upload.wikimedia.org/wikipedia/commons/thumb/5/5b/Lava_%2814273645695%29.jpg/1200px-Lava_%2814273645695%29.jpg"},
+    {maze : "https://upload.wikimedia.org/wikipedia/en/6/67/Homebrew_reaction_diffusion_example_512iter.jpg"},
+    {metal : "https://upload.wikimedia.org/wikipedia/commons/a/aa/Metal.jpg"},
+    {myr : "https://learnmyr.org/img/MYR-Logo.png"},
+    {night_sky : "https://upload.wikimedia.org/wikipedia/commons/2/23/A_Deep_Look_into_a_Dark_Sky.jpg"},
+    {patterned_glass : "https://www.swiftglass.com/assets/iStock_000006016778_Full-500x332.jpg"},
+    {pavement : "https://static.turbosquid.com/Preview/2014/08/01__17_56_46/pavement%201b.jpgba228c77-b71f-4636-a8d9-6da2dde724dbLarger.jpg"},
+    {sand : "https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Sand.jpg/1024px-Sand.jpg"},
+    {snow : "https://upload.wikimedia.org/wikipedia/commons/0/0b/Shadows_on_snow.jpg"},
+    {steel : "https://img.freepik.com/free-photo/silver-metal-texture-background-design_46250-2268.jpg?size=626&ext=jpg"},
+    {stars : "https://upload.wikimedia.org/wikipedia/commons/b/b5/Graphic_patterns_by_Peak_Hora_89.jpg"},
+    {water : "https://upload.wikimedia.org/wikipedia/commons/thumb/3/3b/Mink_Mingle_2017-04-23_%28Unsplash_AlEyuIKuSmc%29.jpg/1200px-Mink_Mingle_2017-04-23_%28Unsplash_AlEyuIKuSmc%29.jpg"},
+    {wicker : "https://upload.wikimedia.org/wikipedia/commons/d/da/Basketwork_wicker_basket_close_up.jpg"},
+    {wood : "https://img.freepik.com/free-photo/wooden-textured-background_53876-14865.jpg?size=626&ext=jpg"},
+    {wool : "https://upload.wikimedia.org/wikipedia/commons/thumb/3/35/Beige_wool_texture.jpg/1200px-Beige_wool_texture.jpg"},
+];

```

Comment:
We shouldn't be relying on outside sources to host these images. We should host them ourselves instead. We can then also optimize the file sizes as we choose.

Reply:
Where would be the best place to host them?